playbooks/active-mentorship.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Active mentorship
+description: Guide for practicing active mentorship between a mentor and mentee
+---
+
+Every junior team member has a dedicated mentor assigned to them for guidance, questions and general technical
+mentorship.
+
+To provide structure into the mentorship, we follow a daily routine sessions in which the mentee can ask their
+mentor questions and discuss given topics.
+
+The routine can be practiced as follows:
+
+-   keep the mentoring session short and focused
+-   maximum time is 30 minutes
+-   no minimum time is specified, meaning the session does not have to be help if there are no topics to discuss
+-   the mentee should prepare specific questions
```

Comment:
+ mentee can track discussed topics, conslusions and ideas inside a dedicated mentor-mentee doc

Reply:
Awesome idea. I suggest that we add this line:
```suggestion
-   the mentee should prepare specific questions
-   the mentee can track discussed topics, conclusions and ideas inside a dedicated mentor-mentee doc, visible to both parties
```

Do you agree, @napravicukod ?